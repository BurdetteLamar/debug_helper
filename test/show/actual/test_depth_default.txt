---
Hash (size=1 message='test_depth_default'):
  Pair 0:
    Key:
      Symbol (size=1):
        to_s: a
        encoding: !ruby/encoding US-ASCII
    Value:
      Hash (size=1):
        Pair 0:
          Key:
            Symbol (size=1):
              to_s: b
              encoding: !ruby/encoding US-ASCII
          Value:
            Hash (size=1):
              Pair 0:
                Key: Symbol :c
                Value: String "ok"
