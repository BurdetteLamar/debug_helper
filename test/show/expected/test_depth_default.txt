---
Hash (message='test_depth_default'):
  Hash#size: 1
  Hash#default: 
  Hash#default_proc: 
  Pair 0:
    Key:
      Symbol:
        Symbol#to_s: a
        Symbol#size: 1
        Symbol#encoding: !ruby/encoding US-ASCII
    Value:
      Hash:
        Hash#size: 1
        Hash#default: 
        Hash#default_proc: 
        Pair 0:
          Key:
            Symbol:
              Symbol#to_s: b
              Symbol#size: 1
              Symbol#encoding: !ruby/encoding US-ASCII
          Value:
            Hash:
              Hash#size: 1
              Hash#default: 
              Hash#default_proc: 
              Pair 0:
                Key: Symbol :c
                Value: String "ok"
